src/nupic/regions/TPRegion.py

```diff
@@ -42,11 +42,13 @@ def _getTPClass(temporalImp):
     return TP10X2.TP10X2
   elif temporalImp == 'tm_py':
     return TP_shim.TPShim
+  elif temporalImp == 'fast_tm_py':
```

Comment:
In some cases we have had "MyClass" (Python) and "FastMyClass" (C++), like the CLA Classifier. In other cases, we have just had "MyClass" in both and referred to them as either "py" or "cpp" versions. My preference for the implementation identifier would be "py" vs "cpp" since that is the only difference. If you think "fast" is better that is ok but it shouldn't have "py" in the name if it is the C++ implementation.